### PR TITLE
consultation2:adding prior consultation scenario

### PR DESCRIPTION
**Pull request by gitcliff**
link:https://github.com/IsantePlus/isanteplus-qaframework/issues/16

### CODE REVIEW

**Comment by gitcliff** [comment]
cc @mozzy11

**Review by sherrif10 — APPROVED** [review]
LGTM. Thanks @gitcliff

File: src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/ConsultationSteps.java [hunk 1]
```diff
@@ -73,4 +73,20 @@ public void redirectedToForms() throws Exception {
...
+
+    @And("On the Ajouter consultation antérieure pop up enter the Date de début and the Date de fin")
+    public void enterDates() throws Exception {
+        clinicianFacingPatientDashboardPage.selectDatesForPreviousConsultation();
+    }
+
+    @And("Click Confirm")
+    public void clickConfirm() throws Exception {
+        patientDashBoardPage = clinicianFacingPatientDashboardPage.clickConfirmPreviousConsultation();
+        Thread.sleep(5000);
```
Comment (mozzy11) [inline src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/ConsultationSteps.java:90]: @gitcliff , you could use  patientDashBoardPage.waitForPage() 
as @sherrif10 reviewed yesterday.
Youll fix it in the next PR

**Comment by mozzy11** [comment]
Thanks @gitcliff 
and thanks @sherrif10 for the review

**Pull request merged**